src/test/java/underdogs/devbie/recommendation/controller/AnswerRecommendationControllerTest.java

```diff
@@ -0,0 +1,88 @@
+package underdogs.devbie.recommendation.controller;
+
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.BDDMockito.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+import static underdogs.devbie.recommendation.RecommendationAcceptanceTest.*;
+import static underdogs.devbie.user.domain.UserTest.*;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+import underdogs.devbie.MvcTest;
+import underdogs.devbie.auth.controller.interceptor.BearerAuthInterceptor;
+import underdogs.devbie.auth.controller.resolver.LoginUserArgumentResolver;
+import underdogs.devbie.recommendation.service.AnswerRecommendationService;
+import underdogs.devbie.user.domain.User;
+
+@WebMvcTest(AnswerRecommendationController.class)
+class AnswerRecommendationControllerTest extends MvcTest {
+
+    @MockBean
+    private BearerAuthInterceptor bearerAuthInterceptor;
+
+    @MockBean
+    private LoginUserArgumentResolver loginUserArgumentResolver;
+
+    @MockBean
+    private AnswerRecommendationService answerRecommendationService;
+
+    @Test
+    void count() throws Exception{
+        getAction("/api/recommendation/answer/1")
+            .andExpect(status().isOk());
+    }
+
+    @Test
+    void createRecommendation() throws Exception {
+        User user = User.builder()
+            .id(1L)
+            .oauthId(TEST_OAUTH_ID)
+            .email(TEST_USER_EMAIL)
+            .build();
+
+        given(bearerAuthInterceptor.preHandle(any(), any(), any())).willReturn(true);
+        given(loginUserArgumentResolver.supportsParameter(any())).willReturn(true);
+        given(loginUserArgumentResolver.resolveArgument(any(), any(), any(), any())).willReturn(user);
+
```

Comment:
```suggestion
@BeforeEach
void setUp() {
        User user = User.builder()
            .id(1L)
            .oauthId(TEST_OAUTH_ID)
            .email(TEST_USER_EMAIL)
            .build();
            
        given(bearerAuthInterceptor.preHandle(any(), any(), any())).willReturn(true);
       given(loginUserArgumentResolver.supportsParameter(any())).willReturn(true);
        given(loginUserArgumentResolver.resolveArgument(any(), any(), any(), any())).willReturn(user);
}
```
테스트에 사용될 인가 부분을 setUp으로 빼는건 어떨까요?